settingengine.go

```diff
@@ -225,3 +227,13 @@ func (e *SettingEngine) SetSRTPReplayProtectionWindow(n uint) {
 func (e *SettingEngine) SetSRTCPReplayProtectionWindow(n uint) {
 	e.replayProtection.SRTCP = &n
 }
+
+// DisableSRTPReplayProtection disables SRTP replay protection.
+func (e *SettingEngine) DisableSRTPReplayProtection(isDisabled bool) {
+	e.disableSRTPReplayProtection = isDisabled
```

Comment:
I think
```suggestion
func (e *SettingEngine) DisableSRTPReplayProtection() {
	e.disableSRTPReplayProtection = true
```
and adding `e.disableSRTPReplayProtection = false` to `SetSRTPReplayProtectionWindow()` simplifies API.